README.md

```diff
@@ -94,31 +94,33 @@ Initializes remix-electron. Returns a promise with a url to load in the browser
 
 Options:
 
-- `serverBuild`: The path to your server build (e.g. `path.join(__dirname, 'build')`), or the server build itself (e.g. required from `@remix-run/dev/server-build`). Updates on refresh are only supported when passing a path string.
+- `serverBuild`: The path to your server build (e.g. `path.join(process.cwd(), 'build')`), or the server build itself (e.g. required from `@remix-run/dev/server-build`). Updates on refresh are only supported when passing a path string.
```

Comment:
Why the change here? I figure `__dirname` or even `import.meta.url` would be a more reliable reference path

Reply:
I think that `__dirname` is not available in ESM, `import.meta.url` should do but we'd need to document both. Happy to make this change if you prefer that but `process.cwd()` should fine I think

Reply:
Yeah it doesn't make a huge difference, I've just been bitten by `process.cwd()` in the past